pkg/client/mapiserver/snapshot.go

```diff
@@ -165,7 +167,7 @@ func ListSnapshot(volName string) error {
 	if code != http.StatusOK {
 		return fmt.Errorf("Server status error: %v", http.StatusText(code))
 	}
-	snapdisk, err := getInfo(body)
+	snapdisk, err := getInfo([]byte(body))
```

Comment:
this is not required. ReadAll method returns []byte which is getting passed to this method.